### ISSUE TITLE
How did the javascript port of the algorithm helped you?

### ISSUE THREAD

**Issue by tricasoft** [davidmchapman/3DContainerPacking#11]
Thanks for your work, it is really appreciated!
This is not an issue, but I couldn't find any way to contact you, sorry for the inconveniences.

I found out from your README that there is a javascript port of the code: https://github.com/keremdemirer/3dbinpackingjs
Did you use something from that repository? Or you just listed it?
I'd like to run the JS version, but it's not working or I'm not aware of how it is working, it's doesn't output the packed boxes and their position in the container, as you can see in the image bellow:
![solution](https://user-images.githubusercontent.com/1433790/47676095-51f6c580-dbc4-11e8-8a84-907d18e1011a.png)

Thanks for any idea!

**Comment by davidmchapman** [comment]
Hello! When I wrote the C# port, I referenced the original C and the earlier JavaScript port. But I did not verify whether the JavaScript port was fully functional. I focused instead on making sure that the C# port passed all the packing tests presented in the original thesis. I also recall running into some issues when I tried to run the JavaScript version, but I can't remember the specifics. I'm sorry, I don't know if this answer is very helpful! :(
